views/address.tmpl

```diff
@@ -38,8 +38,8 @@
                     data-target="address.btns"
                     data-action="click->address#changeView"
                 >
-                    <input id="addr-btn" type="button" class="btn btn_sm btn-active" value="List" name="list">
-                    <input id="addr-btn" type="button" class="btn btn_sm" value="Charts"
+                    <input id="addr-btn" type="button" class="list btn btn_sm btn-active" value="List" name="list">
+                    <input id="addr-btn" type="button" class="chart btn btn_sm" value="Charts"
```

Comment:
I used the id attribute across the three button groups to ensure that in each group only one button is active at time. I could not use the classes because that is what the id selector is helping to modify.

I am aware that you can only select one item with an id selector and that was actually the whole point of using an id selector since only one button should be active at a go.

Reply:
If you use `addr-btn` as a `class` instead of an `id`, I think you can do  

```
$('input.addr-btn').on('click', function(){
    // Remove it from all of them
    $('input.addr-btn').removeClass('btn-active');
    // Reapply it to just this one
    $(this).addClass('btn-active');
});
```
There is probably an even cleaner approach using radio buttons and button groups. Radio buttons have a CSS `:checked` state, so you could even get rid of the javascript if you wanted.

Reply:
I suggest for the UI change we do it in a separate PR because that change might be out of the scope of this PR.

Cool Idea though.

Reply:
```
$('input.addr-btn').on('click', function(){
    // Remove it from all of them
    $('input.addr-btn').removeClass('btn-active');
    // Reapply it to just this one
    $(this).addClass('btn-active');
});
```
This implementation will resort to duplication of the above piece of code for every group of buttons available simply because we should manage the active button of each group individually.

I mean that when I click the charts button, the "zoom" and "group by" buttons should not be affected.

Reply:
If it's just for styling, the quick and dirty way is to just use the siblings selector. 

```
$(this).siblings('input.addr-btn').removeClass('btn-active');
```

Also important to note is that your `input[type=button]` selector will grab any input button on the page, so if somebody puts a button somewhere else, or in the header or footer, there may be unintended consequences.

Reply:
@buck54321 I have dropped the id selector implementation. Kindly help me test the PR when you get the time.